conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
@HeartSaVioR 
If I'm following the code/conversation correctly, this flag essentially changes the IMetricConsumer interface (specifically the value set in the DataPoint class - which is of type Object).  That makes me very uneasy.

Is there a way to make the interface more strongly typed?

Thinking out loud: What about splitting the interface in two: One that does aggregation and one that doesn't, then switching calls based on which interface is implemented?

I'm not opposed to breaking backwards compatibility and bumping up the major version if necessary. In other words I'd rather see a clean design than preserve backwards compatibility with respect to the metrics features.